Formula/dpkg.rb

```diff
@@ -40,6 +40,7 @@ def install
 
     bin.install Dir["#{libexec}/bin/*"]
     man.install Dir["#{libexec}/share/man/*"]
+    (lib/"pkgconfig").install Dir["#{libexec}/lib/pkgconfig/*.pc"]
```

Comment:
This should probably be a symlink to avoid potentially breaking current workflows.